tests/test_models.py

```diff
@@ -9,3 +9,18 @@ def test_region():
 def test_business_area():
     ab = BusinessArea(name="test_name")
     assert str(ab) == 'test_name'
+
+
+def test_user():
+    user = User(username="user_name")
```

Comment:
can we use a factory here?

Reply:
i have created an user factory for other places, but i don't see how it could help in this case